src/screens/InspectionCreate/index.js

```diff
@@ -1,60 +1,120 @@
-import React, { useCallback, useState } from 'react';
-import { SafeAreaView, StyleSheet } from 'react-native';
+import { Constants } from '@monkvision/camera';
+import React, { useCallback, useEffect } from 'react';
+import { useNavigation, useRoute } from '@react-navigation/native';
+import { View, StyleSheet } from 'react-native';
+import { useTheme } from 'react-native-paper';
+import { Loader } from '@monkvision/ui';
+import isEmpty from 'lodash.isempty';
 
-import useScreen from 'screens/InspectionCreate/useScreen';
+import * as names from 'screens/names';
 
-import { Capture, Controls, Constants, useUploads } from '@monkvision/camera';
+import useAuth from 'hooks/useAuth';
+import useSignIn from 'hooks/useSignIn';
+import useCreateInspection, { TASKS_BY_MOD } from './useCreateInspection';
+
+export const SIGHTS_IDS_BY_MOD = {
```

Comment:
Is this `SIGHTS_IDS_BY_MOD` or `SIGHTS_IDS_BY_MODE`?
Same question inside useCreateInspection.

Reply:
Naming is hard. :-( Is mode even a good term for that ?